static/js/components/LearningResourceCard.js

```diff
@@ -120,8 +121,11 @@ export const LearningResourceCard = ({
         <div className="row course-title" onClick={showResourceDrawer}>
           <Dotdotdot clamp={2}>{object.title}</Dotdotdot>
         </div>
-        {object.offered_by ? (
-          <Subtitle content={object.offered_by} label="Offered by - " />
+        {object.offered_by.length ? (
+          <Subtitle
+            content={getPreferredOfferedBy(object.offered_by)}
+            label="Offered by - "
```

Comment:
When a course has offered_by values other than Micromasters, it only displays the first letter:

![Screen Shot 2019-10-18 at 11 46 05 AM](https://user-images.githubusercontent.com/187676/67109009-9eecf680-f19d-11e9-9ac9-ed8bf4dcbc40.png)

![Screen Shot 2019-10-18 at 11 53 25 AM](https://user-images.githubusercontent.com/187676/67109208-03a85100-f19e-11e9-9d4a-d7bf37a86938.png)

Reply:
hmf